server/sonar-server/src/test/java/org/sonar/server/platform/db/EmbeddedDatabaseTest.java

```diff
@@ -50,7 +50,7 @@
   @Rule
   public TemporaryFolder temporaryFolder = new TemporaryFolder();
   @Rule
-  public Timeout timeout = Timeout.seconds(10);
+  public Timeout timeout = Timeout.seconds(30);
```

Comment:
I still would love to change this. It won't change the result of unit tests on any of our machines - but might help users with slow machines to build.

At no cost we can improve the tests.